nodeup/pkg/model/kube_apiserver.go

```diff
@@ -318,6 +352,42 @@ func (b *KubeAPIServerBuilder) buildPod() (*v1.Pod, error) {
 
 	pod.Spec.Containers = append(pod.Spec.Containers, *container)
 
+	if b.Cluster.Spec.EncryptionConfig != nil && *b.Cluster.Spec.EncryptionConfig {
+		sideCar := &v1.Container{
+			Name:  "kms-plugin",
+			Image: "sethpollack/kmsplugin:latest",
```

Comment:
As we discussed, this is OK while we're getting this passing e2e, and I'm trying to figure out how to get autobuilding going :-)